scipy/integrate/_ivp/common.py

```diff
@@ -80,7 +80,8 @@ def select_initial_step(fun, t0, y0, f0, direction, order, rtol, atol):
     direction : float
         Integration direction.
     order : float
-        Method order.
+        Error estimator order. It means that the error controlled by the
+        algorithm is proportional to ``step_size ** (order + 1)`.
```

Comment:
I think you have one ` too many or you're missing one

Reply:
It is a little confusing, but I think it is correct. In numerical ODE solving an algorithm order is meant as its global error (under some "good conditions") and thus the local error is proportional to `h ** (order + 1)`.